sql/core/src/test/scala/org/apache/spark/sql/execution/command/DDLParserSuite.scala

```diff
@@ -246,18 +246,17 @@ class DDLParserSuite extends AnalysisTest with SharedSparkSession {
 
   test("create hive external table - location must be specified") {
     assertUnsupported(
-      sql = "CREATE EXTERNAL TABLE my_tab STORED AS parquet",
+      sql = "CREATE EXTERNAL TABLE my_tab",
```

Comment:
This clearly represents the change: `STORED AS parquet` was required to let the query statement fall into createHiveTable which has been optional. We no longer require it.

Reply:
Do we have to remove this `STORED AS parquet`? The best backward-compatible solution for this test case seems to keep both `STORED AS parquet` and the unsupported exception without any changes on this test case.

Reply:
Could you extend these test cases by adding only, please?

Reply:
So the test wasn't have `STORED AS parquet`, and SPARK-30098 added it. (I guess it was added because it didn't work without that.) 

This change is just reverting to original one, but it's also OK to have another test for having `STORED AS`.